Source/DafnyStandardLibraries/src/DafnyStdLibs/TargetSpecific/ConcurrentInterface-notarget-java-cs-js-go-py.dfy

```diff
@@ -0,0 +1,73 @@
+abstract module DafnyStdLibs.ConcurrentInterface {
```

Comment:
I think this is the right file to put documentation on the types and why their interfaces are what they are (which is counterintuitive for Dafny). Again I'm happy to provide text.